src/EncompassRest.EntityGenerator/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Note, this file was included in the project but not in source control. This fixes that problem.